### PR TITLE
AccName: comp_labelledby tests

### PR DESCRIPTION
**Pull request by cookiecrook**
Closes https://github.com/web-platform-tests/interop-2023-accessibility-testing/issues/35

Some follow-on work is blocked by https://github.com/w3c/accname/issues/209 so additional tests will come in https://github.com/web-platform-tests/interop-2023-accessibility-testing/issues/68

### CODE REVIEW

**Comment by cookiecrook** [comment]
Adding @chrishtr as a reviewer, who may also be able to comment why an unrelated ChromeDriver download failure blocks the required sink-task. Am I reading the logs correctly?

```
tools.wpt.browser.DownloadNotFoundError: No ChromeDriver download found to match browser version 120.0.6062.2 dev
```

**Comment by cookiecrook** [comment]
Recycling CI

File: accname/name/comp_labelledby.html [hunk 1]
```diff
@@ -13,14 +13,36 @@
...
+
 <!--
 
-Todo: test all remaining cases of https://w3c.github.io/accname/#comp_labelledby
+  BLOCKED on https://github.com/w3c/accname/issues/209
+
+  Once that's resolved, it may be worthwhile to add a new ./comp_labelledby_recursion.html file,
+  so that the implementation diffs rolling in don't penalize this Interop 2024 test.
+
+  All remaining cases of https://w3c.github.io/accname/#comp_labelledby
```
Comment (cookiecrook) [inline accname/name/comp_labelledby.html:45]: Reviewers: Some follow-on work is blocked by https://github.com/w3c/accname/issues/209 so additional tests will come separately in https://github.com/web-platform-tests/interop-2023-accessibility-testing/issues/68

File: accname/name/comp_labelledby.html [hunk 2]
```diff
@@ -13,14 +13,36 @@
 
 <p>Tests the <a href="https://w3c.github.io/accname/#comp_labelledby">#comp_labelledby</a> portions of the AccName <em>Name Computation</em> algorithm.</p>
```
Comment (rahimabdi) [inline accname/name/comp_labelledby.html:14]: I've included several related tests for testing hidden not referenced content in ./comp_label we may be worth documenting here.

Comment (cookiecrook) [reply accname/name/comp_labelledby.html:14]: Why here and in `./comp_label.html` rather than in `./comp_hidden_not_referenced.html` with #32

Comment (cookiecrook) [reply accname/name/comp_labelledby.html:14]: To clarify, I'm not saying the file structure needs to stay as-is, but [I asked for feedback on the structure prior to starting](https://github.com/w3c/accname/issues/190) and didn't receive much feedback, and no conflicting opinions.

Comment (cookiecrook) [reply accname/name/comp_labelledby.html:14]: Open to change, but for the sake of [results longevity](https://wpt.fyi/results/accname/name/comp_hidden_not_referenced.html?label=experimental&label=master&aligned&q=label%3Aaccessibility), there should be a compelling reason to rename any tests files and subtests. (click "Show history timeline" to see what I mean)

Comment (rahimabdi) [reply accname/name/comp_labelledby.html:14]: Agreed, definitely not recommending a file structure change; I included name "precedence" tests in `./comp_label.html` so just referring to providing documentation linkages where similar stuff is tested. For example, for `comp_label.html`:

Comment (cookiecrook) [reply accname/name/comp_labelledby.html:14]: Ah got it. Thanks.

File: accname/name/comp_labelledby.html [hunk 3]
```diff
@@ -13,14 +13,36 @@
...
 
-<div role="group" aria-labelledby="h" class="ex" data-expectedlabel="div group label" data-testname="div group explicitly labelledby heading">
-  <h2 id="h">div group label</h2>
+<!--
+
+  Some overlap with tests in:
+  - /accname/name/comp_name_from_content.html
+  - /accname/name/shadowdom/*
+
+-->
```
Comment (cookiecrook) [inline accname/name/comp_labelledby.html:22]:
```suggestion
<!--

  Some overlap with other aria-labelledby tests in:
  - /accname/name/comp_label.html (aria-labelledby / aria-label fallback, etc.)
  - /accname/name/comp_name_from_content.html
  - /accname/name/shadowdom/*

-->
```

File: accname/name/comp_labelledby.html (hunk 2 above)
Comment (cookiecrook) [reply accname/name/comp_labelledby.html:14]: addressed in https://github.com/web-platform-tests/wpt/pull/42522/commits/4803e83d8f62a8336aed1ff83d26cf777243b38d

**Review by rahimabdi — APPROVED** [review]
Looks good!

**Pull request merged**